libse/Dictionaries/OcrFixReplaceList.cs

```diff
@@ -182,16 +182,14 @@ public string FixOcrErrorViaLineReplaceList(string input)
                 {
                     if (s.Contains(from))
                     {
+                        string replaceWith = _beginLineReplaceList[from];
                         if (s.StartsWith(from, StringComparison.Ordinal))
-                            s = s.Remove(0, from.Length).Insert(0, _beginLineReplaceList[from]);
-                        s = s.Replace(". " + from, ". " + _beginLineReplaceList[from]);
-                        s = s.Replace("! " + from, "! " + _beginLineReplaceList[from]);
-                        s = s.Replace("? " + from, "? " + _beginLineReplaceList[from]);
-                        s = s.Replace(". " + Environment.NewLine + from, ". " + Environment.NewLine + _beginLineReplaceList[from]);
-                        s = s.Replace("! " + Environment.NewLine + from, "! " + Environment.NewLine + _beginLineReplaceList[from]);
-                        s = s.Replace("? " + Environment.NewLine + from, "? " + Environment.NewLine + _beginLineReplaceList[from]);
```

Comment:
There will be no `Environment.NewLine` because s is one the element of lines which is `lines = newText.SplitToLines()`